conf/msg_conf/map_msg.conf

```diff
@@ -1706,5 +1706,12 @@
 1503: You've entered a PK Zone.
 1504: You've entered a PK Zone (safe until level %d).
 
+// More Adoption
+1505: The Baby is not level 50 or lower.
```

Comment:
Change number ID, is already being used https://github.com/rathena/rathena/commit/73fe0d26b7f7f3d8bf42cb7b7d459224003144ad